### PR TITLE
Feature/us7672 unique site label

### PR DESCRIPTION
**Pull request by vchtelmakh**
---

### CODE REVIEW

File: src/app/models/pin-label-data.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    this.lastInitial = lastInitial;
+    this.siteName = siteName;
+    this.isHost = isHost;
+    this.isMe = isMe;
+    this.pinType = pinType;
+  }
+
+}
+
+export class PinLabel {
```
Comment (plachmann) [inline src/app/models/pin-label-data.ts]: There is some logic in this model that is probably worthy of a unit test

File: src/app/services/google-map.service.ts [hunk 2]
```diff
@@ -152,10 +153,23 @@ export class GoogleMapService {
...
-    let decDeg = parseInt('' + dec);
-    let decMin = parseInt('' + (dec - decDeg) * 60.0);
-    let decSec = parseInt('' + (dec - decDeg - (decMin / 60.0)) * 3600.0);
+    let decDeg = parseInt('' + dec, 10);
+    let decMin = parseInt('' + (dec - decDeg) * 60.0, 10);
+    let decSec = parseInt('' + (dec - decDeg - (decMin / 60.0)) * 3600.0, 10);
     return `${decSgn}${decDeg}° ${decMin}' ${decSec}"`;
   }
 
+  public getLabelHeightAdjustment(cHeight: number, marker: MapMarker, isHostOrMe: boolean): number {
```
Comment (plachmann) [inline src/app/services/google-map.service.ts]: Add a unit test for this new method.

File: src/app/services/pin-label.service.ts [hunk 3]
```diff
@@ -0,0 +1,87 @@
+import { Injectable, NgZone } from '@angular/core';
+
```
Comment (plachmann) [inline src/app/services/pin-label.service.ts]: Need unit tests for new service

File: src/app/services/google-map.service.ts (hunk 2 above)
Comment (vchtelmakh) [inline src/app/services/google-map.service.ts]: Done.

File: src/app/services/pin-label.service.ts (hunk 3 above)
Comment (vchtelmakh) [inline src/app/services/pin-label.service.ts]: Done.

File: src/app/models/pin-label-data.ts (hunk 1 above)
Comment (vchtelmakh) [inline src/app/models/pin-label-data.ts]: Done.

**Pull request merged**